File: app/src/main/java/com/example/david/lists/widget/MyAppWidgetProvider.java
package com.example.david.lists.widget;

import android.appwidget.AppWidgetManager;
import android.appwidget.AppWidgetProvider;
import android.content.Context;
import android.widget.RemoteViews;

public final class MyAppWidgetProvider extends AppWidgetProvider {

    @Override
    public void onUpdate(Context context, AppWidgetManager appWidgetManager, int[] appWidgetIds) {
        for (int appWidgetId : appWidgetIds) {
            RemoteViews remoteViews = new WidgetRemoteView(context, appWidgetId).updateWidget();
            appWidgetManager.updateAppWidget(appWidgetId, remoteViews);
        }
    }
}

File: app/src/main/java/com/example/david/lists/widget/view/MyAppWidgetProvider.kt
package com.example.david.lists.widget.view

import android.appwidget.AppWidgetManager
import android.appwidget.AppWidgetProvider
import android.content.Context

class MyAppWidgetProvider : AppWidgetProvider() {
    override fun onUpdate(context: Context, appWidgetManager: AppWidgetManager, appWidgetIds: IntArray) {
        for (appWidgetId in appWidgetIds) {
//            val remoteViews = WidgetRemoteView(context, widgetId).updateWidget()
//            appWidgetManager.updateAppWidget(widgetId, remoteViews)
        }
    }
}
